### PR TITLE
Updated Restart and run icon.

### PR DESCRIPTION
**Pull request by tgeorgeux**
## References
I had jumped the gun on merging #8024

## Code changes
Added restart and run Icon instead of FF Icon.

## User-facing changes

Updated Icon

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by saulshanabrook** [comment]
One test to fix here!

https://github.com/jupyterlab/jupyterlab/pull/8091/checks?check_run_id=537381805#step:9:700

**Comment by telamonian** [comment]
I reran the test that was failing locally and it now passes. We might have to wait for a while before the actual CI runs due to the 20 job concurrency limit

**Comment by saulshanabrook** [comment]
Screen Shot 2020-03-30 at 5 30 18 PM

I think the restart and run all icon should look more similar to the restart one...  They are different shapes slightly.

**Comment by telamonian** [comment]
I got inspired and made an alternative restart-and-run icon over at #8136:

image

It should go a bit more harmoniously with the restart icon, while still being visually distinct. Here's what it looks like at real size in context next to the restart icon on the toolbar:

image

**Comment by tgeorgeux** [comment]
Screen Shot 2020-04-01 at 4 28 13 PM

Reverted to the Fast Forward Icon, but with the cannon Material Icon Outline used here.

File: packages/ui-components/style/icons/toolbar/fast-forward.svg [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```
Comment (saulshanabrook) [inline packages/ui-components/style/icons/toolbar/fast-forward.svg:1]: Is this used anywhere else? Can it be deleted now?

**Comment by joelostblom** [comment]
Thanks for adding this feature in the recent previous PR, it's quite convenient!

I was going to open an issue for changing the icon from fast forward to restart arrow + fast forward or restart + play when I came across the two already existing issues. I think it is is easier to understand the additive icon suggested by @tgeorgeux versus the merged icon suggested by @telamonian and it is more aesthetically pleasing with the play button centered both horizontally and vertically, but this is just a personal opinion, I don't have any icon design experience.

This is how I believe I would naively interpret the icons (bitmap mockup):

image

I think a single play button inside a restart circle arrow can convey "run multiple" as it is clear there are space restrictions, but there is of course the chance that it is misinterpreted as "Restart and run one".

This is with horizontal flip of the restart arrow as I noticed was discussed previosuly:

image

**Comment by jasongrout** [comment]
FYI, VSCode python extension has a counter-clockwise restart icon:
Screen Shot 2020-05-02 at 5 59 05 AM

**Comment by isabela-pf** [comment]
I’m unclear why the original PR was merged too early, but I would advocate for the icon originally suggested in [this comment](https://github.com/jupyterlab/jupyterlab/pull/8024#issuecomment-604092781) by @tgeorgeux (and reiterated by @joelostblom in the last of the list above). I agree the icon should have the arrow going counter-clockwise to accurately reflect the standards we follow with the rest of our material icons. I think this is good to go.

**Comment by jweill-aws** [comment]
There hasn't been any activity on this pull request in over a year. As part of an effort to clean up this repository's pull requests, I'm closing this request. If you would still like to merge this code in, please reopen the request and resolve any outstanding merge conflicts. All of the code and comments will remain in place. Thank you for being part of the JupyterLab community!

**Pull request closed without merging**